src/Management/src/EndpointBase/Info/Contributor/GitInfoContributor.cs

```diff
@@ -15,9 +16,10 @@ public class GitInfoContributor : AbstractConfigurationContributor, IInfoContrib
     {
         private const string GITSETTINGS_PREFIX = "git";
         private const string GITPROPERTIES_FILE = "git.properties";
+        private const string DATETIME_OUTPUT_FORMAT = "yyyy-MM-ddTHH:mm:ssZ";
 
-        private static readonly DateTime BaseTime = new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc);
-
+        private readonly List<string> DATETIME_INPUT_KEYS = new List<string> { "time" };
+        
```

Comment:
```suggestion
        private static readonly List<string> DATETIME_INPUT_KEYS = new List<string> { "time" };
```
Extremely minor stuff here